tests/e2e/specs/psi-activation.test.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * WordPress dependencies
+ */
+import { switchUserToAdmin, visitAdminPage, activatePlugin, deactivatePlugin } from '@wordpress/e2e-test-utils';
+import { resetSiteKit } from '../utils/reset';
+
+describe( 'PageSpeed Insights Activation', () => {
+	beforeEach( async() => {
+	} );
+
+	afterEach( async() => {
+	} );
+
+	it( 'Dashboard should display Activate PageSpeed Insights CTA', async() => {
+
+		await resetSiteKit();
+		await activatePlugin( 'e2e-tests-auth-plugin' );
+		await visitAdminPage( 'admin.php', 'page=googlesitekit-dashboard' );
+		await page.waitForSelector( '.googlesitekit-cta__title' );
+		const welcomeTitle = await page.$x(
+			'//h3 [ contains( @class,"googlesitekit-cta__title") and contains( text(), "Activate PageSpeed Insights.")]'
+		);
+		expect( welcomeTitle.length ).not.toEqual( 0 );
+	} );
+
+	it( 'Setup PSI button should lead you to the activation page', async() => {
+		const setupButton = await page.$x(
```

Comment:
Could also use `toMatchElement` here and then for the button click use `page.click()` with the same selector.